CONTRIBUTING.txt

```diff
@@ -379,6 +379,27 @@ To rebuild a full clean documentation, run:
     make clean
     make html
 
+Gallery
+~~~~~~~
+
+If you are contributing an example to the
+`gallery <https://scikit-image.org/docs/dev/auto_examples/>`_ (or editing an
+existing one), write in
+`reStructuredText <https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html>`_.
+Indeed, the gallery is a
+`Sphinx-Gallery <https://sphinx-gallery.github.io/stable/index.html>`_ one.
+
+Refer to the above to build (all) the docs, and check how your edits render at
+``scikit-image/doc/build/html/auto_examples/`` (navigate to the file you have
+added or changed).
+
+When adding an example, visit also
```

Comment:
also visit

Reply:
I find there is a slight difference in meanings, but I'm not sure: I wrote 'visit also `index.html`' because the contributor was just told to *visit* another page (visit A, visit also B); 'also visit' sounds like the previous instruction would entail an action different from visiting a web page (e.g., run make html, also visit the resulting web pages).